### ISSUE TITLE
When saving a query it is not listed after save

### ISSUE THREAD

**Issue by davecoggins** [flipboxfactory/craft-salesforce#1]
Hi

When creating a query I can preview result which all looks correct but when I save it, it is not listed in query list view.

However when I look at the salesforce_queries table it is in there.

We are interested in using this plugin for a new project, it looks really good, do you have a timeline for production release? Am I right in thinking this is currently a release candidate?

Kind regards
Dave

**Comment by nateiler** [comment]
Hi Dave,

We have experience this before which is likely caused by an external dependency.  Can you try to update our 'Ember' package?

`composer require flipboxfactory/craft-ember:^2.5.1`

As for production; we're treating this as a production package (versioning, deprecations, etc and it's currently in the wild across various projects.  It's not in the Craft Plugin store because we haven't produced ample documentation nor determined if adding it is appropriate.

**Comment by davecoggins** [comment]
Hi Natelier,

Thanks for looking into this for me and for also claryfing the status of the plugin.

I have tried updating your ember package as described but it doesn't seem to have a fixed the issue. Here is the output from composer:

`./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Nothing to install or update
Writing lock file
Generating optimized autoload files
Carbon 1 is deprecated, see how to migrate to Carbon 2.
https://carbon.nesbot.com/docs/#api-carbon-2
    You can run './vendor/bin/upgrade-carbon' to get help in updating carbon and other frameworks and libraries that depend on it.`

In package.json file the following line has been added:

`"flipboxfactory/craft-ember": "^2.5.1",`
